src/Swashbuckle.AspNetCore.SwaggerGen/Application/SwaggerGenOptionsExtensions.cs

```diff
@@ -1,9 +1,9 @@
 using System;
 using System.Collections.Generic;
 using System.Xml.XPath;
-using Microsoft.AspNetCore.Mvc.ApiExplorer;
 using Microsoft.OpenApi.Models;
 using Swashbuckle.AspNetCore.SwaggerGen;
+using ApiDescriptionType = Microsoft.AspNetCore.Mvc.ApiExplorer.ApiDescription;
```

Comment:
What's the reason for aliasing `ApiDescription` here - is there a conflict? Would be good to understand the need for this.

Reply:
Yes, we introduced a conflict when we created the `IDocumentProvider` interface -- that namespace conflicts with this type name.